src/widgetastic/utils.py

```diff
@@ -610,3 +611,18 @@ def __get__(self, o, t):
 
     def __repr__(self):
         return 'Ignore({!r})'.format(self.wt_class)
+
+
+def resolve_version_pick(func):
+    @wraps(func)
+    def wrapped(*args, **kwargs):
+        version = func.browser.product_version
+        wargs = []
+        for arg in args:
+            wargs.append(arg.pick(version) if isinstance(arg, VersionPick) else arg)
+        wkwargs = kwargs.copy()
+        for arg, value in wkwargs.items():
+            wkwargs[arg] = arg.pick(version) if isinstance(arg, VersionPick) else arg
+        r = func(*wargs, **wkwargs)
+        return r
+    return wrapped
```

Comment:
You don't need to write this function as it is. Use `ConstructorResolvable` class instead. Like in src/widgetastic/widget.py#L36